plugin/trino-hive-hadoop2/bin/common.sh

```diff
@@ -76,12 +76,30 @@ function cleanup_hadoop_docker_containers() {
     cleanup_docker_containers "${DOCKER_COMPOSE_LOCATION}"
 }
 
-function termination_handler(){
+function termination_handler() {
     set +e
     cleanup_docker_containers "$@"
     exit 130
 }
 
+# Check that all arguments are the names of non-empty variables.
+function require_env() {
+    ( # Use a subshell to avoid errexit and preserve xtrace
+        set +x
```

Comment:
any particular reason to do `set +x`? document.

Reply:
My goal was to provide simple error messages without the clutter of all the checks.

If I make it take just one variable, I think I can remove that.

Reply:
fine. Just maybe add a comment why `set +x`.